tests/testthat/test-shinytest2-tm_g_pp_patient_timeline.R

```diff
@@ -0,0 +1,19 @@
+app_driver_tm_g_km <- function() {
+
+}
+
+testthat::test_that("e2e - tm_g_km: Module initializes in teal without errors and produces plot output.", {
+  skip_if_too_deep(5)
+  app_driver <- app_driver_tm_g_km()
+  app_driver$expect_no_shiny_error()
+  app_driver$expect_no_validation_error()
+
+  testthat::expect_match(
+    app_driver$get_attr(
+      app_driver$active_module_element("myplot-plot_main > img"),
```

Comment:
Same reasoning as here https://github.com/insightsengineering/teal.modules.clinical/pull/1132/files#r1576318046

IMHO it would be better if we can go module by module, so we can focus on one module and catch such things.
```suggestion
      app_driver$active_module_element("patient_timeline_plot-plot_main > img"),
```